docs/wiki/archive/Magnetometer.mdx

```diff
@@ -133,7 +145,7 @@ We can use this fact to check the orientation of the Mag after it is attached to
 
 We need to know the 'north' direction of the Earth's magnetic field, approximately, at our location, so that we can point the nose of the quad kind of directly into that magnetic field vector when we go looking at the data returned by our Mag.
 
-First, get your phone out, open it's Compass app, and mark a line on the bench or ground or whatever that points due North/South. Then draw a second line that takes into account the magnetic Declination for your location. In my case, some 13 degrees east of true North. That declination-adjusted line is the direction that Magnetic North will be in.
+First, get your phone out, open it's Compass app, and mark a line on the bench or ground or whatever that points due North/South. Then draw a second line that takes into account the magnetic Declination for your location. In my case, some 13 degrees east of the geographic North. That declination-adjusted line is the direction that Magnetic North will be in.
 
```

Comment:
I should have said, "That declination-adjusted line points in the direction of Magnetic North" ?